### ISSUE TITLE
Failed Installation using MELPA, Emacs 26.3.50

### ISSUE THREAD

**Issue by jason-morgan** [org-roam/org-roam#1229]
### Description

Installation using MELPA is failing, reporting `Cannot open load file: No such file or directory, ol` when trying to compile 4 files. I have verified that the files exist in the directory. Messages from `*Compile-Log*` below.

```
Compiling no file at Fri Oct 30 11:11:38 2020
Leaving directory ‘/home/jason/.emacs.d/elpa/org-roam-20201030.623’

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-buffer.el at Fri Oct 30 11:11:38 2020
Entering directory ‘/home/jason/.emacs.d/elpa/org-roam-20201030.623/’
org-roam-buffer.el:38:1:Error: Cannot open load file: No such file or directory, ol

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-capture.el at Fri Oct 30 11:11:38 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-compat.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-completion.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-dailies.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-db.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-dev.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-doctor.el at Fri Oct 30 11:11:39 2020

In org-roam-doctor--replace-link:
org-roam-doctor.el:218:28:Warning: reference to free variable
    ‘org-link-bracket-re’

In org-roam-doctor--replace-link-keep-label:
org-roam-doctor.el:233:28:Warning: reference to free variable
    ‘org-link-bracket-re’

In org-roam-doctor--remove-link:
org-roam-doctor.el:250:26:Warning: reference to free variable
    ‘org-link-bracket-re’

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-faces.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-graph.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-link.el at Fri Oct 30 11:11:39 2020
org-roam-link.el:37:1:Error: Cannot open load file: No such file or directory, ol

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-macs.el at Fri Oct 30 11:11:39 2020

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam-protocol.el at Fri Oct 30 11:11:39 2020
org-roam-protocol.el:39:1:Error: Cannot open load file: No such file or directory, ol

Compiling file /home/jason/.emacs.d/elpa/org-roam-20201030.623/org-roam.el at Fri Oct 30 11:11:39 2020
org-roam.el:59:1:Error: Cannot open load file: No such file or directory, ol

```

**Comment by myshevchuk** [comment]
You also need fresh Org-mode from MELPA.

**Comment by d12frosted** [comment]
You might need these instructions: https://orgmode.org/elpa.html (depending on the approach you use to install packages).

**Comment by jason-morgan** [comment]
Ok. Thanks for the help. To clarify, the `org-mode` included with Emacs 26.3.50 is too out of date? I have wanted to upgrade to 27.1, but there is a TRAMP bug that has prevented me from doing so.

**Comment by myshevchuk** [comment]
I believe the one shipped with 27.1 may also be out of date. Or will be soon. But you don't need to upgrade yet if you don't want to. Just install Org-mode from MELPA. It's almost bleeding edge and pretty stable. Follow the instructions in the link provided by @d12frosted to properly install it. People often run into trouble having two versions of Org-mode installed and then incorrectly loading the built-in one leads to all kinds of errors and unpredictable behaviour.

**Comment by jason-morgan** [comment]
Thank you @myshevchuk and @d12frosted for the help. Seems to have solved my issue.
